contrib/docker/docker-compose.yml

```diff
@@ -6,16 +6,22 @@ version: '3'
 services:
 
   synapse:
-    image: matrixdotorg/synapse:latest
+    image: docker.io/matrixdotorg/synapse:latest
     # Since snyapse does not retry to connect to the database, restart upon
     # failure
     restart: unless-stopped
     # See the readme for a full documentation of the environment settings
     environment:
       - SYNAPSE_SERVER_NAME=my.matrix.host
+      - SYNAPSE_REPORT_STATS=no
```

Comment:
The default "no" has just been moved now, I am still not sure people will take a serious look at this.